src/web/lib/components/AppBackground/index.js

```diff
@@ -6,9 +6,11 @@ import './index.scss';
 
 export const AppBackground = ({ theme }) => {
   const colors = {};
-  Object.keys(theme.colors).forEach(key => {
-    colors[key] = colorToCSS(theme.colors[key]);
-  });
+  if (theme) {
```

Comment:
if the `theme` object is falsey here, what does that mean for L15 below, where we try and access `colors.toolbar` and `colors.accentcolor`? Wouldn't both of those be `undefined` since `colors` is an empty object?

Reply:
@pdehaan afaik there's a cascade in firefox and the theme only requires two values for a custom theme:
https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/theme/update

Reply:
Oh, disregard previous, i see what you're saying...file a separate issue?